modules/ols-thinking-model-generates-delineator-prompt.adoc

```diff
@@ -0,0 +1,13 @@
+// This module is used in the following assemblies:
+// troubleshoot/ols-troubleshooting-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ols-thinking-model-generates-delineator-prompt_{context}"]
+= Thinking model generates delineator prompt
+
+In some cases, thinking or reasoning models produce output that has delineators, such as `THOUGHT` or `reasoning`, to separate the thinking process from the model output that the {ols-long} Service uses to answer your question. 
+
+The {ols-long} Service does not configure the delineator behavior or produce the delineators in the output. The delineator feature is a model configuration setting. Typically, you can disable the feature using one of the following methods:
+
+* Append your question prompt with the specific keyword that the model requires to toggle the delineator feature off. For example, `/nothink`.
```

Comment:
need to specify that this is dependent on the model and the model's documentation should indicate if this works or not.